docs/storage_controller.md

```diff
@@ -0,0 +1,147 @@
+# Storage Controller
+
+## Concepts
+
+The storage controller sits between the user and the pageserver, and handles the details of mapping tenants to pageserver tenant shards.
```

Comment:
nit: "storage controller sits between the user and the pageserver" makes it sound like the storage controller is a proxy which is not the case

Reply:
changed s/user/administrative API clients/